README.md

```diff
@@ -61,6 +61,39 @@ The element manages matching and entering the appropriate affordance states.  Id
 [affordance="collapse"] h2 { color: blue; }
 ```
 
+### Shadow Parts for affordances
+The element is composed of an internal Shadow DOM containing 2 slots and exposing several parts which are exposed for author styling.  The structure of these parts is 
+
+```
+::part(tab-bar)
+   ::part(tab-list)
+     /* tabListSlot */
+::part(content-panels)
+  /* default slot */
+```
+
+By default all contents are slotted into the `::part
```

Comment:
Should this be `::part` with a closing `? Or is the part missing here?

Reply:
Good catch, it should be slot anyways.. Fixed